.gitignore

```diff
@@ -2,3 +2,4 @@
 /adminer*.php
 /editor*.php
 /vendor/
+*.DS_Store
```

Comment:
Put this into your `core.excludesfile` Git config file.

Reply:
Thanks. Done